init/config.js

```diff
@@ -61,4 +61,19 @@ module.exports = {
     calendarId: "c_9c8sc215f8vopmjuj4ogamd7go@group.calendar.google.com",
     discordWebhookUrl: null
   },
+  opportunity: {
+    level: "Beginner",
+    commitmentHoursPerWeek: 2,
+    description: "test opportunity description",
+    expectations: [
+      {
+        "id": 0,
+        "expectation": "test expectation"
+      }
+    ],
+    isHidden: true,
+    roleCategory: "Development",
+    roleType: "Back-End Developer"
+  },
+  opportunityTitles : ["test opportunity", "test second opportunity"]
```

Comment:
This looks a bit unnatural. Can we create 2 opportunities instead of the same opportunity, but 2 titles.